phpBB/styles/prosilver/theme/colours.css

```diff
@@ -314,6 +314,41 @@ p.post-notice {
 	background-image: none;
 }
 
+/*	colours and backgrounds for mentions.css */
```

Comment:
Remove the tab should be a space

Reply:
I've made that the same as for other modules, changed that.
The same for your comment below.